searchlib/src/vespa/searchlib/queryeval/fake_search.h

```diff
@@ -17,6 +17,7 @@ class FakeSearch : public SearchIterator
     std::string             _term;
     FakeResult                   _result;
     uint32_t                     _offset;
+    bool                         _unpacked;
```

Comment:
consider using last docid unpacked instead of bool to avoid entanglement with seek